package.json

```diff
@@ -44,26 +44,26 @@
     "@glimmer/tracking": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.24.0",
+    "ember-cli": "~3.28.6",
     "ember-cli-addon-docs": "^4.0.2",
     "ember-cli-addon-docs-yuidoc": "^1.0.0",
-    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^2.0.0",
     "ember-cli-deploy-git": "^1.3.4",
     "ember-cli-deploy-git-ci": "^1.0.1",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.1",
+    "ember-cli-terser": "^4.0.2",
     "ember-data": "~3.28.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^1.0.0",
     "ember-page-title": "^6.0.3",
     "ember-qunit": "^5.1.1",
-    "ember-resolver": "^8.0.2",
-    "ember-source": "^3.24.0",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~3.28.0",
```

Comment:
[question] Just to confirm - should this version have the ~ now instead of the ^?

Reply:
Good catch. Ember has it using `tilda` (safer) but now that you ask I'm just going to reset this one back. I don't think ember-source made a difference anyway.

Reply:
Reset `ember-source` back just to be safe.

Reply:
Nvmd, ended up having to bring `"ember-source": "~3.28.0",` back to match ember-ui.